homeassistant/components/overkiz/climate/atlantic_electrical_heater_with_adjustable_temperature_setpoint.py

```diff
@@ -154,3 +156,16 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
         await self.executor.async_execute_command(
             OverkizCommand.SET_TARGET_TEMPERATURE, temperature
         )
+
+    @property
+    def extra_state_attributes(self) -> dict[str, Any]:
+        """Set extra attributes such as regulation mode (none, standby, increase)."""
+        states = self.device.states
+        attributes: dict[str, Any] = {}
+        if (state := states[OverkizState.CORE_REGULATION_MODE]) and state.value_as_str:
+            attributes[ATTR_REGULATION_MODE] = state.value_as_str
+        if (
+            state := states[OverkizState.CORE_DEROGATED_TARGET_TEMPERATURE]
+        ) and state.value_as_float:
+            attributes[ATTR_DEROGATED_TARGET_TEMPERATURE] = state.value_as_float
```

Comment:
I will need to double check with another core contributors, but in general adding extra attributes is not recommended. 

Could we make a sensor or binary sensor out of these states? It should be relatively easy to add them due to our use of EntityDescriptions.

Reply:
Indeed, that's was very simple ! Thanks for the tip